runtime/server/metrics_views.go

```diff
@@ -346,18 +346,18 @@ func buildFilterClauseForCondition(cond *runtimev1.MetricsViewFilter_Cond, exclu
 	var operatorPrefix string
 	var conditionJoiner string
 	if exclude {
-		operatorPrefix = "NOT"
-		conditionJoiner = "AND"
+		operatorPrefix = " NOT "
```

Comment:
How different is this function vs the one in timeseries_api.go? We should try and maintain a single filter to where clause converter function.

Reply:
There're not different functionally, it's only a matter of time I start refactoring that and merge implementation where possible.